instrumentation/all/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-all
 
+### v0.24.0 / 2022-04-29
+
+* ADDED: Adds instrumentation for rdkafka 
```

Comment:
The dependency for `opentelemetry-instrumentation-rdkafka` seems to be missing from https://github.com/open-telemetry/opentelemetry-ruby/blob/main/instrumentation/all/opentelemetry-instrumentation-all.gemspec